karapace/utils.py

```diff
@@ -5,17 +5,19 @@
 See LICENSE for details
 """
 from functools import partial
-from kafka.client_async import KafkaClient, MetadataRequest
 from urllib.parse import urljoin
 
 import datetime
 import decimal
 import json as jsonlib
+import kafka.client_async
 import logging
 import requests
+import time
 import types
 
 log = logging.getLogger("KarapaceUtils")
+BLACKLIST_DURATION = 60
```

Comment:
What is the unit (s or ms)?
Why 60, is it based in what?

Reply:
It is seconds, and based on nothing specifically.... I just wanted the DNS errors to be silenced but not gone forever, so a minute seemed like a good idea